maintainers/maintainer-list.nix

```diff
@@ -3033,6 +3033,12 @@
     githubId = 1436960;
     name = "Christoph Hrdinka";
   };
+  hrhino = {
```

Comment:
I think the convention is to put this into a separate commit titled `maintainers: add hrhino` (added #96666)

Reply:
Done; thanks.

Reply:
also pushed a newer version because, why not.

Reply:
yikes. thanks.